docs/developer.md

```diff
@@ -95,10 +95,10 @@ make deploy-openshift [IMAGE_TAG_BASE=<your-registry>/backstage-operator]
 ```
 
 ### Modifying the API definitions
-If you are editing the API definitions, generate the manifests such as CRs or CRDs using:
+If you are editing the API definitions, regenerate the manifests and bundle using:
```

Comment:
To cover all the deployment options we offer, I would say something like:
If you are editing the API definitions, make sure you :
- run 'make install' before deploying operator with 'make deploy'
- regenerate the manifests and bundle if you plan to deploy operator with OLM using: ...